src/styles/mainMenu.scss

```diff
@@ -169,3 +169,31 @@ h1 {
         border-radius: 8px;
     }
 }
+
+.close-button {
+    position: absolute;
+    top: 10px;
+    right: 10px;
+    z-index: 1000;
+    background: transparent;
+    border: none;
+    color: #fff;
+    font-size: 28px;
+    width: 40px;
+    height: 40px;
+    line-height: 40px;
+    text-align: center;
+    cursor: pointer;
+    border-radius: 3px;
+    padding: 0;
+    transition:
+        background 0.15s,
```

Comment:
In the CSS, I have variables `$transition-duration` and `$main-font` as well as the accent colors. Look for them in the other CSS files and use them here instead of duplicating the values